wpilibc/src/main/native/cpp/smartdashboard/MechanismLigament2d.cpp

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#include "frc/smartdashboard/MechanismLigament2d.h"
+#include <wpi/raw_ostream.h>
```

Comment:
Don't need a raw_ostream.h include here, just need to include cstdio for snprintf.